patroni/api.py

```diff
@@ -176,13 +179,31 @@ def do_POST_failover(self):
         member = request.get('member', None)
         cluster = self.server.patroni.ha.dcs.get_cluster()
         status_code = 503
```

Comment:
Shouldn't we return 500 internal error instead of 503 service unavailable for exceptions?